azure-pipelines.yml

```diff
@@ -0,0 +1,165 @@
+trigger:
+  - main
+
+resources:
+  - repo: self
+
+variables:
+  tag: '$(Build.BuildId)'
+  ABS_ACCOUNT_NAME: ciimportexportlib
+  ABS_CONTAINER_NAME: testing
+  AZURE_RESOURCE_GROUP: ci-import-export-lib
+  AZURE_SERVICE_PRINCIPAL: http://ci-import-export-lib
+  AZURE_SERVICE_PRINCIPAL_TENANT: 9b85ee6f-4fb0-4a46-8cb7-4dcc6b262a89
+  SNOWFLAKE_DATABASE: KEBOOLA_DB_IMPORT_EXPORT
+  SNOWFLAKE_HOST: kebooladev.snowflakecomputing.com
+  SNOWFLAKE_PORT: 443
+  SNOWFLAKE_USER: KEBOOLA_DB_IMPORT_EXPORT
+  SNOWFLAKE_WAREHOUSE: DEV
+  SYNAPSE_SERVER_NAME: ci-testing-php-db-import-export
+
+
+stages:
+  - stage: Build
+    displayName: Build and test
+    jobs:
+      - job: Build
+        timeoutInMinutes: 100
+        displayName: Build
+        pool:
+          vmImage: 'ubuntu-latest'
+        steps:
```

Comment:
tyjo, ta práce s envama je tam teda ultra debilní :( Neví někdo v @keboola/team-sre jestli to nejde udělat nějak líp?

Reply:
Možná by bylo docela cool k tomu mít nějaký tooling, ať nevymýšlíme v každém buildu znova kolo...

Reply:
Těma envama myslí `echo "##vso[task.setvariable variable=`?

Reply:
Oboje - jak to, že jsou tam ty vso tak to, že každý ten skript má 20 ručně vypsaných env

Reply:
https://github.com/keboola/php-db-import-export/pull/39/files/8a4ebc864fa3479d63ceb858d346092ab975e55f#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5R56

i 

https://github.com/keboola/php-db-import-export/pull/39/files/8a4ebc864fa3479d63ceb858d346092ab975e55f#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5R87

Reply:
![image](https://user-images.githubusercontent.com/642928/104703862-2a926180-5718-11eb-8c0c-f4d493de3382.png)
![image](https://user-images.githubusercontent.com/642928/104703877-30884280-5718-11eb-8c4d-a01991dc3b4f.png)

To je úplně ultrazbytečná práce a ještě k tomu náchylná na typo :(

Reply:
Jak by sis to představoval? Fakt si nejsem jistej, že něco jako "vem všechny envy a importni je do pipeline" je správnej postup. Díky tomu explicitnímu mapování přesně víš, co tam bude. Totéž vlastně i pro ten `##vso` helper. Může na to bejt nějaká helper fce, které předáš název envu, ale to budeš muset tu helper funkci odněkud importovat...

Reply:
Tak když uděláš typo tak ti to někde failne, no.
Zajímavý je ten nápad vytlačit to ze scriptu co teď generuju jako env_export (https://github.com/keboola/php-db-import-export/pull/39#discussion_r558009757) nevím jestli to bude fungovat

Reply:
Zkusil sem https://github.com/keboola/php-db-import-export/pull/39/commits/265568f6df01b4e25ff6835276b0f92d3c9e2d97 myslím, že by to mělo fungovat

Reply:
Fičí to, trošku míň řádků aspoň :)